src/ChargeDriftModels/ADLChargeDriftModel/ADLChargeDriftModel.jl

```diff
@@ -159,39 +159,88 @@ function ADLChargeDriftModel{T,M,N,TM}(chargedriftmodel::ADLChargeDriftModel{<:A
 end
 
 
-ADLChargeDriftModel{T}(args...; kwargs...) where {T <: SSDFloat} = ADLChargeDriftModel(args..., T=T, kwargs...)
-function ADLChargeDriftModel(configfilename::Union{Missing, AbstractString} = missing; T::Type=Float32, material::Type{<:AbstractDriftMaterial} = HPGe,
-                             temperature::Union{Missing, Real}= missing, phi110::Union{Missing, Real} = missing)::ADLChargeDriftModel{T}
-
-    if ismissing(configfilename) configfilename = joinpath(get_path_to_example_config_files(), "ADLChargeDriftModel/drift_velocity_config.yaml") end
-    if !ismissing(temperature) temperature = T(temperature) end  #if you give the temperature it will be used, otherwise read from config file
-
-    config = parse_config_file(configfilename)
-
-    #mu0 in m^2 / ( V * s )
-    #beta dimensionless
-    #E0 in V / m
-    #mun in m^2 / ( V * s )
-
-    e100mu0::T  = config["drift"]["velocity"]["parameters"]["e100"]["mu0"]
-    e100beta::T = config["drift"]["velocity"]["parameters"]["e100"]["beta"]
-    e100E0::T   = config["drift"]["velocity"]["parameters"]["e100"]["E0"]
-    e100mun::T  = config["drift"]["velocity"]["parameters"]["e100"]["mun"]
-    e111mu0::T  = config["drift"]["velocity"]["parameters"]["e111"]["mu0"]
-    e111beta::T = config["drift"]["velocity"]["parameters"]["e111"]["beta"]
-    e111E0::T   = config["drift"]["velocity"]["parameters"]["e111"]["E0"]
-    e111mun::T  = config["drift"]["velocity"]["parameters"]["e111"]["mun"]
-    h100mu0::T  = config["drift"]["velocity"]["parameters"]["h100"]["mu0"]
-    h100beta::T = config["drift"]["velocity"]["parameters"]["h100"]["beta"]
-    h100E0::T   = config["drift"]["velocity"]["parameters"]["h100"]["E0"]
-    h111mu0::T  = config["drift"]["velocity"]["parameters"]["h111"]["mu0"]
-    h111beta::T = config["drift"]["velocity"]["parameters"]["h111"]["beta"]
-    h111E0::T   = config["drift"]["velocity"]["parameters"]["h111"]["E0"]
-
-    e100 = VelocityParameters{T}(e100mu0, e100beta, e100E0, e100mun)
-    e111 = VelocityParameters{T}(e111mu0, e111beta, e111E0, e111mun)
-    h100 = VelocityParameters{T}(h100mu0, h100beta, h100E0, 0)
-    h111 = VelocityParameters{T}(h111mu0, h111beta, h111E0, 0)
+
+ADLChargeDriftModel{T}(args...; kwargs...) where {T <: SSDFloat} = ADLChargeDriftModel(args...; T=T, kwargs...)
+
+const defaultADLfile = joinpath(get_path_to_example_config_files(), "ADLChargeDriftModel/drift_velocity_config.yaml")
+function ADLChargeDriftModel(configfilename::AbstractString = defaultADLfile; kwargs...)
+    ADLChargeDriftModel(parse_config_file(configfilename); kwargs...)
+end
```

Comment:
If no config file is provided, the default ADLChargeDriftModel parameters are loaded as basis.